src/runtime/components/navigation/Tabs.vue

```diff
@@ -131,6 +131,7 @@ export default defineComponent({
 
     watch(() => props.modelValue, (value) => {
       selectedIndex.value = value
+      calcMarkerSize(value)
```

Comment:
Would you mind opening a different PR for this fix? It's way easier to keep track in the release note this way.

Reply:
> Would you mind opening a different PR for this fix? It's way easier to keep track in the release note this way.

Yes, I open #524